binder/src/test/java/io/grpc/binder/SecurityPoliciesTest.java

```diff
@@ -345,6 +345,7 @@ public PackageInfoBuilder setSignatures(Signature... signatures) {
       return this;
     }
 
+    @SuppressWarnings("deprecation")
```

Comment:
What's going on here?

Reply:
> I need to look over this more, but is the test case design documented anywhere? Since other languages will need to implement this as well, we need a textual description. That would also help me when reviewing.
> 
> I'm concerned with how this is getting shoe-horned into the interop code. We can maybe discuss more when I get a better high-level view, but are we planning to have any interop tests cross-language? If not, then it probably isn't worth all this orca special-casing. If it is, then I'd have to consider our alternatives.

Thank you. I share you concerns. 
I was planning to add this cross language, it would be needed to do the internal CI tests as we planned. Doc is something like: https://github.com/grpc/grpc/pull/29632